security/build_process.adoc

```diff
@@ -8,68 +8,60 @@
 :toc: macro
 :toc-title:
 :prewrap!:
-:sourcedir: ../
 
 toc::[]
 
 [[security-build-build-once-deploy-everywhere]]
 == Build Once, Deploy Everywhere
 
-In a containerized environment, the software build process is the stage in the
-life cycle where application code is integrated with the required runtime
-libraries. Managing this build process is key to securing the software stack.
-Adhering to a "build once, deploy everywhere" philosophy ensures that the
-product of the build process is exactly what is deployed in production.
+In a container environment, the software build process is the stage in the life
+cycle where application code is integrated with the required runtime libraries.
+Managing this build process is key to securing the software stack. Adhering to a
+"build once, deploy everywhere" philosophy ensures that the product of the build
+process is exactly what is deployed in production.
 
-It is also important to maintain the immutability of your containers; in other
-words, do not patch running containers, but instead rebuilt and redeploy them.
+It is also important to maintain the immutability of your containers. You should
+not patch running containers, but rebuild and redeploy them.
 
 Using {product-title} as the standard platform for container builds enables you
 to guarantee the security of the build environment as well.
```

Comment:
The above sentence seems out of place, like it is leading to a point, but is instead the end of the section. Is there something missing? Should is be somewhere else?